source/installation-guide/securing_api.rst

```diff
@@ -5,28 +5,97 @@
 Securing the Wazuh API
 ======================
 
-By default, the communications between the Wazuh Kibana App and the Wazuh API are not encrypted. It is highly recommended that you secure the Wazuh API by following the steps below:
+By default, the communications between the Wazuh Kibana plugin and the Wazuh API are not encrypted. In addition to this, the default user is "foo" and the password is  "bar". For this reason, it is highly recommended that you secure the Wazuh API and change the default user and password. The following steps will help you to achieve it:
 
-1. Enable HTTPS:
+#. Enable HTTPS:
 
-  In order to enable HTTPS, you can generate your own certificate or generate it automatically by using the script ``/var/ossec/api/scripts/configure_api.sh``.
-  
-  .. note::
-    This script allows you to change the port used by the Wazuh API to handle the incoming HTTP requests. The port 55000 is used by default.
+    In order to enable HTTPS, you can generate your own certificate or generate it automatically by using the script ``/var/ossec/api/scripts/configure_api.sh``.
 
-    **Change the default credentials:**
+        #. Option a: Generate the certificate automatically by using the script.
 
-    The ``configure_api.sh`` script allows you to change the API's user. If you did not use the script you can still change it as follows:
-    
-    .. code-block:: console
+          You need to execute the script ``/var/ossec/api/scripts/configure_api.sh`` and follow the script steps. It will ask you for all the necessary parameters and it will create the certificate using the data provided in the interactive process. The script will restart the Wazuh API automatically.
 
-      # cd /var/ossec/api/configuration/auth
-      # node htpasswd -Bc -C 10 user myUserName
-      
-    By default, you can access the Wazuh API by typing user "foo" and password "bar".
- 
-You will then need to restart the ``wazuh-api`` and ``wazuh-manager`` services for the change to take effect.
+        #. Option b: Use your own certificate. The file ``/var/ossec/api/configuration/config.js`` contains the section:
```

Comment:
In the file ``/var/ossec/api/configuration/config.js`` you will find the following section:

Reply:
Done: https://github.com/wazuh/wazuh-documentation/commit/061e85fd2f28632303170e52adb708af05dad86c

Reply:
Done: https://github.com/wazuh/wazuh-documentation/commit/ce4ef2ab0d2a44b6eb8a6054345bbf3b375e553d